.github/workflows/ci.yml

```diff
@@ -82,6 +82,7 @@ jobs:
   ci:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
+    container: ${{ matrix.container }}
```

Comment:
I don't see where you set a key `container` in your changes below. Why is this needed?

Reply:
Right. This is left over from an earlier attempt to select the image. Alpak CI is using `ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=` for this.
```suggestion
```